test/edu/ucsb/nceas/metacat/restservice/v2/MNResourceHandlerTest.java

```diff
@@ -0,0 +1,382 @@
+package edu.ucsb.nceas.metacat.restservice.v2;
+
+import org.apache.wicket.protocol.http.mock.MockHttpServletRequest;
+import org.apache.wicket.protocol.http.mock.MockHttpServletResponse;
+import org.apache.wicket.protocol.http.mock.MockHttpSession;
+import org.apache.wicket.protocol.http.mock.MockServletContext;
+import org.dataone.service.types.v1.Identifier;
+import org.dataone.service.types.v1.Session;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+
+import static org.mockito.ArgumentMatchers.any;
+
+import java.util.List;
+import java.util.Vector;
+
+import edu.ucsb.nceas.LeanTestUtils;
+import edu.ucsb.nceas.metacat.dataone.MNodeService;
+
+
+
+/**
+ * A test class of the MNResourceHandler class
+ * @author tao
+ *
+ */
+
+public class MNResourceHandlerTest {
+
+    /**HTTP Verb GET*/
+    protected static final byte GET = 1;
+    /**HTTP Verb POST*/
+    protected static final byte POST = 2;
+    /**HTTP Verb PUT*/
+    protected static final byte PUT = 3;
+    /**HTTP Verb DELETE*/
+    protected static final byte DELETE = 4;
+    /**HTTP Verb HEAD*/
+    protected static final byte HEAD = 5;
```

Comment:
Field 'HEAD' is never used

Reply:
Weird. I added a comment before. This one can be used in the future. I think we have the code to use `HEAD`.